less/search.less

```diff
@@ -34,10 +34,9 @@
 	max-height: 500px;
 	overflow-x: hidden;
 	overflow-y: auto;
+	z-index: @zindex-tooltip + 1;
```

Comment:
:thinking: what's wrong with the inherited z-index?

Reply:
@julianlam Tooltip has a bigger index than quick-search. After hovering on the search engine shows up and the cursor begins to go haywire.